client/state/initial-state.js

```diff
@@ -82,7 +83,13 @@ export function persistOnChange( reduxStore, serializeState = serialize ) {
 			.catch( ( setError ) => {
 				debug( 'failed to set redux-store state', setError );
 			} );
-	}, SERIALIZE_THROTTLE, { leading: false, trailing: true } ) );
+	}, SERIALIZE_THROTTLE, { leading: false, trailing: true } );
+
+	if ( global.window ) {
```

Comment:
Interesting, I normally check with `typeof window !== 'undefined'`. Anyone know where we're defining global? global isn't available in a standard browser env. Maybe @ockham ?